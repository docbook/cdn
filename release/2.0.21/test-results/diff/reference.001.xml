<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: reference.001</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="reference">
         <header deltaxml:deltaV2="A=B" class="reference-titlepage">
            <h1>Unit Test: reference.001</h1>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span class="refentrytitle">
                        <a href="#minimal">Minimal</a>
                     </span>
                     <span class="refpurpose"> - Test of a minimal RefEntry</span>
                  </li>
                  <li>
                     <span class="refentrytitle">
                        <a href="#lessminimal">Also Minimal Title</a>
                     </span>
                     <span class="refpurpose"> - Another test of a minimal RefEntry</span>
                  </li>
                  <li>
                     <span class="refentrytitle">
                        <a href="#evenlessminimal">Fred</a>
                     </span>
                     <span class="refpurpose"> - Test of a minimal RefEntry</span>
                  </li>
               </ul>
            </div>
         </div>
         <article deltaxml:deltaV2="A" id="minimal" class="refentry">
            <div class="refnamediv">
               <h2>Name</h2>
               <p>
                  <span class="refname">Minimal</span>
                  <span class="refpurpose">
                     <span class="refpurpose-sep"> — </span>
                     <span class="refpurpose-text">Test of a minimal RefEntry</span>
                     <span class="refpurpose-punc">.</span>
                  </span>
               </p>
            </div>
            <div class="refsect1">
               <div class="refsection-titlepage">
                  <h2>Description</h2>
               </div>
               <p>This is a minimal RefEntry.</p>
            </div>
         </article><div deltaxml:deltaV2="B" class="content">
            <article id="minimal" class="refentry">
               <div class="content">
                  <div class="refnamediv">
                     <h2>Name</h2>
                     <p>
                        <span class="refname">Minimal</span>
                        <span class="refpurpose">
                           <span class="refpurpose-sep"> — </span>
                           <span class="refpurpose-text">Test of a minimal RefEntry</span>
                           <span class="refpurpose-punc">.</span>
                        </span>
                     </p>
                  </div>
                  <div class="refsect1">
                     <div class="refsection-titlepage">
                        <h2>Description</h2>
                     </div>
                     <div class="content">
                        <p>This is a minimal RefEntry.</p>
                     </div>
                  </div>
               </div>
            </article>
            <article id="lessminimal" class="refentry">
               <div class="refentry-separator">
                  <hr/>
               </div>
               <div class="content">
                  <div class="refnamediv">
                     <h2>Name</h2>
                     <p>
                        <span class="refname">Also Minimal</span>
                        <span class="refpurpose">
                           <span class="refpurpose-sep"> — </span>
                           <span class="refpurpose-text">Another test of a minimal RefEntry</span>
                           <span class="refpurpose-punc">.</span>
                        </span>
                     </p>
                  </div>
                  <div class="refsect1">
                     <div class="refsection-titlepage">
                        <h2>Description</h2>
                     </div>
                     <div class="content">
                        <p>This is a minimal RefEntry.</p>
                     </div>
                  </div>
               </div>
            </article>
            <article id="evenlessminimal" class="refentry">
               <div class="refentry-separator">
                  <hr/>
               </div>
               <div class="content">
                  <div class="refnamediv">
                     <h2>Name</h2>
                     <p>
                        <span class="refname">Less Minimal</span>, <span class="refname">Even Less Minimal</span>
                        <span class="refpurpose">
                           <span class="refpurpose-sep"> — </span>
                           <span class="refpurpose-text">Test of a minimal RefEntry</span>
                           <span class="refpurpose-punc">.</span>
                        </span>
                     </p>
                  </div>
                  <div class="refsect1">
                     <div class="refsection-titlepage">
                        <h2>Description</h2>
                     </div>
                     <div class="content">
                        <p>This is a minimal RefEntry.</p>
                     </div>
                  </div>
               </div>
            </article>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="lessminimal" class="refentry">
            <div class="refentry-separator">
               <hr/>
            </div>
            <div class="refnamediv">
               <h2>Name</h2>
               <p>
                  <span class="refname">Also Minimal</span>
                  <span class="refpurpose">
                     <span class="refpurpose-sep"> — </span>
                     <span class="refpurpose-text">Another test of a minimal RefEntry</span>
                     <span class="refpurpose-punc">.</span>
                  </span>
               </p>
            </div>
            <div class="refsect1">
               <div class="refsection-titlepage">
                  <h2>Description</h2>
               </div>
               <p>This is a minimal RefEntry.</p>
            </div>
         </article><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="evenlessminimal" class="refentry">
            <div class="refentry-separator">
               <hr/>
            </div>
            <div class="refnamediv">
               <h2>Name</h2>
               <p>
                  <span class="refname">Less Minimal</span>, <span class="refname">Even Less Minimal</span>
                  <span class="refpurpose">
                     <span class="refpurpose-sep"> — </span>
                     <span class="refpurpose-text">Test of a minimal RefEntry</span>
                     <span class="refpurpose-punc">.</span>
                  </span>
               </p>
            </div>
            <div class="refsect1">
               <div class="refsection-titlepage">
                  <h2>Description</h2>
               </div>
               <p>This is a minimal RefEntry.</p>
            </div>
         </article>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>