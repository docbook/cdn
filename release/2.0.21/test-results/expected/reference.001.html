<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: reference.001</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="reference"><header class="reference-titlepage"><h1>Unit Test: reference.001</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span class="refentrytitle"><a href="#minimal">Minimal</a></span><span class="refpurpose"> - Test of a minimal RefEntry</span></li><li><span class="refentrytitle"><a href="#lessminimal">Also Minimal Title</a></span><span class="refpurpose"> - Another test of a minimal RefEntry</span></li><li><span class="refentrytitle"><a href="#evenlessminimal">Fred</a></span><span class="refpurpose"> - Test of a minimal RefEntry</span></li></ul></div></div><article id="minimal" class="refentry"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsect1"><div class="refsection-titlepage"><h2>Description</h2></div><p>This is a minimal RefEntry.</p></div></article><article id="lessminimal" class="refentry"><div class="refentry-separator"><hr /></div><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsect1"><div class="refsection-titlepage"><h2>Description</h2></div><p>This is a minimal RefEntry.</p></div></article><article id="evenlessminimal" class="refentry"><div class="refentry-separator"><hr /></div><div class="refnamediv"><h2>Name</h2><p><span class="refname">Less Minimal</span>, <span class="refname">Even Less Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsect1"><div class="refsection-titlepage"><h2>Description</h2></div><p>This is a minimal RefEntry.</p></div></article></article><script src="../../resources/js/prism.js"></script></body></html>